docs/integrations/aa/candide.md

```diff
@@ -0,0 +1,325 @@
+```jsx
+// Sign
+const domain = {
+  chainId: process.env.CHAIN_ID,
+  verifyingContract: smartAccount.safe4337ModuleAddress,
+};
+const types = SafeAccount.EIP712_SAFE_OPERATION_TYPE;
+// formate according to EIP712 Safe Operation Type
+const { sender, ...userOp } = userOperation;
+const safeUserOperation = {
+  ...userOp,
+  safe: userOperation.sender,
+  validUntil: BigInt(0),
+  validAfter: BigInt(0),
+  entryPoint: smartAccount.entrypointAddress,
+};
+const signature = await guardianSigner.signTypedData(domain, types, safeUserOperation);
+const formatedSig = SafeAccount.formatEip712SignaturesToUseroperationSignature([ownerPublicAddress], [signature]);
+userOperationRecovery.signature = signature;
+
+// Submit
+const sendUserOpResponseRecovery =
+    await guardianSmartAccount.sendUserOperation(
+      userOperationRecovery,
+      process.env.BUNDLER_URL
+);
+```
+
+#### Monitor UserOp
+```jsx
+// Wait for receipt
+const userOpReceiptResultRecovery = await sendUserOpResponseRecovery.included();
+
+console.log(userOpReceiptResultRecovery);
+```
+
+### Finalize the Recovery
+After the grace period is over, you can finalize the recovery
+
+#### Prepare the Finalization UserOp
+```jsx
+const finalizeRecoveryMetaTx = srm.createFinalizeRecoveryMetaTransaction(
+    smartAccount.accountAddress
+);
+
+let userOperationFinalizeRecovery = await guardianSmartAccount.createUserOperation(
+    [finalizeRecoveryMetaTx],
+    process.env.JSON_RPC_NODE_PROVIDER,
+    process.env.BUNDLER_URL
+);
+```
+
+#### Sponsor the Gas
+```jsx
+// Add gas sponsorship info using paymaster
+userOperationFinalizeRecovery = await paymaster.createSponsorPaymasterUserOperation(
+    userOperationRecovery,
+    process.env.BUNDLER_URL
+);
+```
+
+#### Sign and Submit
+```jsx
+// Sign userOperation
+const domain = {
+  chainId: process.env.CHAIN_ID,
+  verifyingContract: smartAccount.safe4337ModuleAddress,
+};
+
+const types = SafeAccount.EIP712_SAFE_OPERATION_TYPE;
+
+// formate according to EIP712 Safe Operation Type
+const { sender, ...userOp } = userOperation;
+const safeUserOperation = {
+  ...userOp,
+  safe: userOperation.sender,
+  validUntil: BigInt(0),
+  validAfter: BigInt(0),
+  entryPoint: smartAccount.entrypointAddress,
+};
+
+const signature = await guardianSigner.signTypedData(domain, types, safeUserOperation);
+const formatedSig = SafeAccount.formatEip712SignaturesToUseroperationSignature([ownerPublicAddress], [signature]);
+
+userOperationRecovery.signature = signature;
+
+
+// Submit userOperation
+const sendUserOperationResponseRecovery = await guardianSmartAccount.sendUserOperation(
+    userOperationRecovery,
+    process.env.BUNDLER_URL
+);
+```
+
+#### Monitor UserOperation
+
+```jsx
+const userOperationReceiptResultRecovery = await sendUserOperationResponseRecovery.included();
+
+console.log(userOperationReceiptResultRecovery);
+```
+That's it! You've successfully added a Guardian capable of recovering an account with a Google Account using Lit.
```

Comment:
wasn't the Guardian added in guide 1, whereas this guide was about executing the recovery process itself?

Reply:
That is the case, seems like Marc just left it and I hadn't noticed this error. Changed to:

> That's it! You've successfully recovered an account with a Google Account using Lit.